### DISCUSSION TITLE
documentation mistakes

### DISCUSSION THREAD

**Discussion by Guustaaf** [micropython/micropython#11668] [RP2040 / Pico] [answered]
who do I talk to about mistakes in the documentation?
specifically, this page https://docs.micropython.org/en/latest/rp2/quickref.html#timers
suggests to pass microseconds to the timer object whereas in reality it is milliseconds.

**Comment by dlech** [comment, marked as answer]
You don't have to talk to anyone, you can fix it yourself. 😄 

The source is here: https://github.com/micropython/micropython/blob/master/docs/rp2/quickref.rst

If you are logged in to GitHub, there is an edit button you can click to edit the file in your web browser and then make a pull request to propose the change.

**Comment by jimmo** [comment]
As @dlech said, PR's are definitely welcome. We also have a "Documentation Issue" category on the "new issue" page here https://github.com/micropython/micropython/issues/new/choose
